components/pricing_modal/starter_disclaimer_cta.tsx

```diff
@@ -2,22 +2,20 @@
 // See LICENSE.txt for license information.
 import React from 'react';
 import {useIntl} from 'react-intl';
-import {useDispatch} from 'react-redux';
+import {useDispatch, useSelector} from 'react-redux';
 import styled from 'styled-components';
 
+import {Product} from '@mattermost/types/cloud';
+
+import {getCloudProducts} from 'mattermost-redux/selectors/entities/cloud';
+
 import {openModal, closeModal} from 'actions/views/modals';
 import CloudUsageModal from 'components/cloud_usage_modal';
 
-import {ModalIdentifiers} from 'utils/constants';
-import {FileSizes} from 'utils/file_utils';
+import {CloudProducts, ModalIdentifiers} from 'utils/constants';
+import {fallbackStarterLimits, asGBString} from 'utils/limits';
 import {t} from 'utils/i18n';
 
-const STARTER_MESSAGES_LIMIT = 10000;
-const STARTER_FILE_STORAGE_LIMIT = 10;
-const STARTER_BOARDS_VIEW_LIMIT = 5;
-const STARTER_BOARD_CARDS_LIMIT = 500;
-const STARTER_INTEGRATIONS_LIMIT = 5;
```

Comment:
Is the fallback set to these values? Will it still show correct the starter values if they are in the Professional plan?

Reply:
The fallback is set to the same values. In doing this, I also found that the fallback values incorrectly had integrations set to `10` (https://github.com/mattermost/mattermost-webapp/pull/10574/files#diff-e17a240792fb4a5c6097a1f132b8a01b4a7c6199e2d5205fb21d0f2862deee28L33) , so also took the opportunity to true that up. The fallback is hard coded, so it will still show the correct starter values if they are in the Professional plan.